draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/styles.scss

```diff
@@ -4,30 +4,49 @@
 @import "~@kaizen/deprecated-component-library-helpers/styles/type";
 @import "../../decision-tokens.scss";
 
+$message-icon-size: 1.25rem;
+$validation-message-padding: 0.5625rem;
```

Comment:
Do we not have base tokens in this size? Are these definitely unique values?

Also, the icon size we definitely want rem and not em (not saying it's wrong, just wondering if we'd want it to scale)?

Reply:
These seem to be very specific to this message to ensure there's enough spacing but not so much that it forces a larger response message onto multiple lines. There was a specific ticket this harkens back to in the jira ticket, so I've opted to create a one off scss variable since its not reused outside of this.

Re: em sizing, I've avoided using this since it doesn't appear as a common measurement in our styles.